README.md

```diff
@@ -107,7 +107,21 @@ Edge runtime or middleware is enabled it will also generate edge functions for m
 
 ## Manually installing the Next.js Runtime
 
-The Next.js Runtime installs automatically for new Next.js sites on Netlify. You can also install it manually like this:
+The Next.js Runtime installs automatically for new Next.js sites on Netlify. You can also install it manually in the
+following ways:
+
+### From the UI (Recommended):
+
+You can go to:
+
+```text
+https://app.netlify.com/teams/slug/plugins/@netlify/plugin-nextjs/install
+```
```

Comment:
@Hrishikesh-K I believe this should work for a direct installation link, and it does not require the user to update a slug for their team.

https://app.netlify.com/plugins/@netlify/plugin-nextjs/install